src/site/content/en/learn/css/animations/index.md

```diff
@@ -178,7 +178,7 @@ which accepts four number values: `x1`, `y1`, `x2`, `y2`.
 
 ```css
 .my-element {
-	animation-timing-function: cubic-bezier(.42, 0, .58, 1);
+  animation-timing-function: cubic-bezier(.42, 0, .58, 1);
```

Comment:
Could you please let me to leading zero the numbers? I think readability is much important.

```css
.my-element {
  animation-timing-function: cubic-bezier(0.42, 0, 0.58, 1);
}
```